--- xmame-0.6.7/qtmame-2.0.4/qtmame/Makefile.in.orig	Fri Apr 18 09:52:02 2003
+++ xmame-0.6.7/qtmame-2.0.4/qtmame/Makefile.in	Fri Apr 18 09:53:05 2003
@@ -233,13 +233,13 @@
 qtmame_SOURCES=configi.cpp  ventanai.cpp  main.cpp  config.cpp ventana.cpp
 SOURCES = configi.cpp ventanai.cpp main.cpp 
 INTERFACES = config.ui ventana.ui 
-TRANSLATIONS = qtmame_us.ts qtmame_es.ts 
+TRANSLATIONS = qtmame_us.ts qtmame_es.ts qtmame_pl.ts 
 KDE_OPTIONS = qtonly
 qtmame_LDADD = $(LIB_QPE) $(LIB_QT) $(LIBSOCKET)
 
 SUBDIRS = docs 
 
-EXTRA_DIST = main.cpp ventana.ui ventanai.cpp ventanai.h configi.h config.ui configi.cpp imagenes.h qtmame3.png qtmame.png qtmame.pro qtmame_es.ts qtmame_us.ts qtmame_us.qm qtmame_es.qm Qtmame.desktop qtmame-us bkground.png listado020 catver.ini 
+EXTRA_DIST = main.cpp ventana.ui ventanai.cpp ventanai.h configi.h config.ui configi.cpp imagenes.h qtmame3.png qtmame.png qtmame.pro qtmame_es.ts qtmame_us.ts qtmame_pl.ts qtmame_us.qm qtmame_es.qm qtmame_pl.qm Qtmame.desktop qtmame-us bkground.png listado020 catver.ini 
 
 ####### kdevelop will overwrite this part!!! (end)############
 
--- xmame-0.6.7/qtmame-2.0.4/qtmame/qtmame.pro.orig	Fri Apr 18 09:52:27 2003
+++ xmame-0.6.7/qtmame-2.0.4/qtmame/qtmame.pro	Fri Apr 18 09:55:23 2003
@@ -4,9 +4,9 @@
   MOC_DIR = .moc
   OBJECTS_DIR = .obj
 }
-TRANSLATIONS    = qtmame_us.ts \
-				qtmame_es.ts
-
+TRANSLATIONS    = qtmame-2.0.4/qtmame/qtmame_us.ts \
+				qtmame-2.0.4/qtmame/qtmame_es.ts \
+						qtmame-2.0.4/qtmame/qtmame_pl.ts
 FORMS	= ventana.ui config.ui  
 TEMPLATE	=app
 CONFIG	+= qt warn_on release
--- xmame-0.6.7/qtmame-2.0.4/qtmame/main.cpp.orig	Fri Apr 18 09:55:49 2003
+++ xmame-0.6.7/qtmame-2.0.4/qtmame/main.cpp	Fri Apr 18 09:56:08 2003
@@ -14,7 +14,7 @@
 
         // translation file for application strings
         QTranslator myapp( 0 );
-        myapp.load( QString( "/usr/local/share/qtmame/qtmame_" ) + QTextCodec::locale(), "." );
+        myapp.load( QString( "/usr/share/qtmame/qtmame_" ) + QTextCodec::locale(), "." );
         app.installTranslator( &myapp );
 
       ventanai m;
