diff -ur xmame-0.86.orig/src/unix/video-drivers/svgalib.c xmame-0.86/src/unix/video-drivers/svgalib.c
--- xmame-0.86.orig/src/unix/video-drivers/svgalib.c	2004-08-26 03:44:50.000000000 +0200
+++ xmame-0.86/src/unix/video-drivers/svgalib.c	2004-09-04 21:46:36.872682816 +0200
@@ -154,7 +154,9 @@
 	vga_setmode(video_mode);
 	gl_setcontextvga(video_mode);
 	text_mode = FALSE;
+#ifdef __CPU_i386
 	set_tweaked_mode();
+#endif
 
 	if (video_modeinfo.flags & IS_MODEX)
 		update_function=1;
@@ -279,6 +281,7 @@
 				my_modeinfo->width, my_modeinfo->height, my_modeinfo->colors);
 	}
 
+#ifdef __CPU_i386
 	if (use_tweak && depth == 8 && vga_hasmode(G320x200x256) &&
 			vga_hasmode(G320x240x256))
 	{
@@ -322,6 +325,7 @@
 					tweaked_modes[i].width, tweaked_modes[i].height);
 		}
 	}
+#endif /* ifdef __CPU_i386 */
 
 	if (best_score == 0)
 	{
