--- xmame-0.65.1/makefile.unix~	Sun Mar  9 23:15:20 2003
+++ xmame-0.65.1/makefile.unix	Sun Mar  9 23:16:53 2003
@@ -334,13 +334,13 @@
 INSTALL_USER =         root
 INSTALL_GROUP =        bin
 
-INSTALL_PROGRAM_DIR =	$(INSTALL) -d    -o $(INSTALL_USER) -g $(INSTALL_GROUP) -m  755
-INSTALL_MAN_DIR =	$(INSTALL) -d    -o $(INSTALL_USER) -g 0   -m  755
-INSTALL_DATA_DIR =	$(INSTALL) -d    -o $(INSTALL_USER) -g $(INSTALL_GROUP) -m  755
-INSTALL_PROGRAM =	$(INSTALL) -c -s -o $(INSTALL_USER) -g $(INSTALL_GROUP) -m  555
-INSTALL_PROGRAM_SUID =	$(INSTALL) -c -s -o $(INSTALL_USER) -g $(INSTALL_GROUP) -m 4555
-INSTALL_MAN =		$(INSTALL) -c    -o $(INSTALL_USER) -g $(INSTALL_GROUP) -m  444
-INSTALL_DATA =		$(INSTALL) -c    -o $(INSTALL_USER) -g $(INSTALL_GROUP) -m  644
+INSTALL_PROGRAM_DIR =	$(INSTALL) -d
+INSTALL_MAN_DIR =	$(INSTALL) -d
+INSTALL_DATA_DIR =	$(INSTALL) -d
+INSTALL_PROGRAM =	$(INSTALL) -c
+INSTALL_PROGRAM_SUID =	$(INSTALL) -c
+INSTALL_MAN =		$(INSTALL) -c
+INSTALL_DATA =		$(INSTALL) -c
 
 
 ###########################################################################
